gnovm/pkg/gnolang/machine.go

```diff
@@ -761,6 +761,21 @@ func (m *Machine) EvalStatic(last BlockNode, x Expr) TypedValue {
 	return res[0]
 }
 
+func (m *Machine) emitCallArg(call *CallExpr, x Expr, staticType Type) {
+	// these functions are the only onces in the language were they could be
+	// called from a const context
+	var builtins = []string{"len", "cap", "real", "imag", "complex"}
```

Comment:
Figuring out if its a relevant builtin function by comparing strings is okay because 

1. Builtin functions are not alias-able
2. They are not assign-able
3. We do not allow shadowing of builtins